pkg/api/azenvtypes.go

```diff
@@ -80,7 +80,7 @@ var (
 		VnetCNIWindowsPluginsDownloadURL:     "https://kubernetesartifacts.azureedge.net/azure-cni/" + AzureCniPluginVerWindows + "/binaries/azure-vnet-cni-singletenancy-windows-amd64-" + AzureCniPluginVerWindows + ".zip",
 		ContainerdDownloadURLBase:            "https://storage.googleapis.com/cri-containerd-release/",
 		CSIProxyDownloadURL:                  "https://kubernetesartifacts.azureedge.net/csi-proxy/" + DefaultWindowsCsiProxyVersion + "/binaries/csi-proxy-" + DefaultWindowsCsiProxyVersion + ".tar.gz",
-		WindowsProvisioningScriptsPackageURL: "https://kubernetesartifacts.azureedge.net/aks-engine/windows/provisioning/signedscripts-" + DefaultWindowsProvisioningScriptsPackageVersion + ".zip",
+		WindowsProvisioningScriptsPackageURL: "https://akseashpackages.blob.core.windows.net/scripts/signedscripts-" + DefaultWindowsProvisioningScriptsPackageVersion + ".zip",
```

Comment:
maybe a pain to change at this point, but I like `akseashartifacts` better

Reply:
on the same note, the path `/windows/provisioning` also seems more appropriated.

Reply:
Update